app/views/categories/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h3>Categories</h3>
+
+<ul>
```

Comment:
Yes, we thought it would be better design to include the form on the same page along with the list of categories. If we had separated them, I think it would have been too much clicking around just to submit a new category.